jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -88,6 +94,9 @@
 
     /** Arguments to pass into main. */
     @Nullable private List<String> Cmd;
+
+    /** Network ports the container exposes. */
+    @Nullable private SortedMap<String, Map<?, ?>> ExposedPorts;
```

Comment:
It might make sense to have this be a custom object that stores a list. That way, there doesn't have to be any logic in `getContainerExposedPorts`.